internal/pkg/db/db.go

```diff
@@ -26,7 +26,9 @@ const (
 
 	// Data
 	EventsCollection          = "event"
+	DeletedEventsCollection   = "gc:" + EventsCollection
 	ReadingsCollection        = "reading"
+	DeletedReadingsCollection = "gc:" + ReadingsCollection
```

Comment:
I suggest putting these constants in the actual db/redis package since they aren't common to all DB platforms.